pyzx/simplify.py

```diff
@@ -356,3 +356,98 @@ def clifford_iter(g: BaseGraph[VT,ET]) -> Iterator[Tuple[BaseGraph[VT,ET],str]]:
     yield from pivot_iter(g)
     yield from id_iter(g)
     yield from spider_iter(g)
+
+
+def is_graph_like(g):
+    # checks that all spiders are Z-spiders
```

Comment:
Should be a documentation string """using these quotes""" instead of a regular comment.